Makefile

```
@@ -1,26 +1,20 @@
-PACKAGES=$(shell go list ./... | grep -v '/simulation')
 DOCKER := $(shell which docker)
 DOCKER_BUF := $(DOCKER) run --rm -v $(CURDIR):/workspace --workdir /workspace bufbuild/buf
-VERSION := $(shell echo $(shell git describe --tags) | sed 's/^v//')
-COMMIT := $(shell git log -1 --format='%H')
 
-# TODO: Update the ldflags with the app, client & server names
-ldflags = -X github.com/cosmos/cosmos-sdk/version.Name=NewApp \
-	-X github.com/cosmos/cosmos-sdk/version.ServerName=pooltoyd \
-	-X github.com/cosmos/cosmos-sdk/version.ClientName=pooltoycli \
-	-X github.com/cosmos/cosmos-sdk/version.Version=$(VERSION) \
-	-X github.com/cosmos/cosmos-sdk/version.Commit=$(COMMIT) 
 
-BUILD_FLAGS := -ldflags '$(ldflags)'
+###############################################################################
+###                     Install & Clean                                     ###
+###############################################################################
 
 all: install
 
 install: go.sum
-		go install $(BUILD_FLAGS) ./cmd/pooltoyd
+	@echo "--> installing pooltoy"
+	@go install -v -mod=readonly ./...
```

```suggestion
	@go install -v -mod=readonly ./cmd/pooltoyd
```

Should we keep the build flags as well?

---

yea true, will put those back in